src/components/vr_cooperation/src/command/support_service_request.cc

```diff
@@ -53,6 +53,7 @@ void SupportServiceRequest::Execute() {
   message_.set_rpc(vr_hmi_api::SUPPORT_SERVICE);
   message_.set_rpc_type(vr_hmi_api::REQUEST);
   message_.set_correlation_id(parent()->GetNextCorrelationID());
+  parent()->RegisterRequest(message_.correlation_id(), this);
   parent()->SendMessageToHMI(message_);
```

Comment:
Forgot to add_observer(), You can add_observer in SendMessageToHMI of BaseGpbRequest and use base method here.